pkg/controller/controller.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 const (
-	tprName = "etcd-cluster.coreos.com"
+	TPRName = "etcd-cluster.coreos.com"
```

Comment:
Can we move this to pkg/util/constants/ ? Otherwise client will import the whole controller package.